alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
@mbj Alternatively I could append something like this to the existing header value:

```
default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self';
```